test/PropertyService.test.js

```diff
@@ -0,0 +1,28 @@
+const expect = require("chai").expect;
+const sinon = require("sinon");
+const { cloneDeep } = require("lodash");
+const { PropertyService } = require("../entities/PropertyService");
+
+describe("PropertyService", () => {
+  const properties = cloneDeep(PropertyService.properties);
+  beforeEach(() => {
+    PropertyService.properties = properties;
+  });
+  it("should load contain 40 properties by default", () => {
```

Comment:
I've the same thought here as with the $50 fine - I think we should brainstorm how to better handle this